src/main/kotlin/com/tolkiana/taco/tacoservice/ProductController.kt

```diff
@@ -9,22 +9,22 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 class ProductController(private val resourceLoader: ResourceLoader) {
 
-    @GetMapping("/tacos", produces = arrayOf(MediaType.APPLICATION_JSON_VALUE))
+    @GetMapping("/tacos", produces = [(MediaType.APPLICATION_JSON_VALUE)])
```

Comment:
Nice! Kotlin learning from Swift how to create arrays 😜
🤙

Reply:
If only Swift could turn JSON into Objects 🐤